mne/io/eeglab/eeglab.py

```diff
@@ -663,66 +663,11 @@ def read_events_eeglab(eeg, event_id=None, event_id_func='strip_to_integer',
                          uint16_codec=uint16_codec)['EEG']
 
     annotations = _read_annotations_eeglab(eeg)
-    types = annotations.description
-    latencies = annotations.onset
-
-    if "boundary" in types and "boundary" not in event_id:
-        warn("The data contains 'boundary' events, indicating data "
-             "discontinuities. Be cautious of filtering and epoching around "
-             "these events.")
-
-    if len(types) < 1:  # if there are 0 events, we can exit here
-        logger.info('No events found, returning empty stim channel ...')
-        return np.zeros((0, 3))
-
-    if (latencies < -1).any():
-        raise ValueError('At least one event sample index is negative. Please'
-                         ' check if EEG.event.sample values are correct.')
-    if (latencies == -1).any():
```

Comment:
I don't fully understand what happened with this part of the code. Either, the warning should be raised or removed from the tests.